src/mmw/apps/modeling/tasks.py

```diff
@@ -30,6 +31,9 @@
 
 DRB = 'drb'
 
+RWD_HOST = os.environ.get('RWD_HOST', 'localhost')
+RWD_PORT = os.environ.get('RWD_PORT', '5000')
```

Comment:
Provided some default values because not everyone has these variables provisioned yet.